### ISSUE TITLE
script_runs()/script_compiles() change arrayref argument. 

### ISSUE THREAD

**Issue by stphnlyd** [plicease/Test-Script#9]
For example, 
```perl
my @cmd = ['foo', 'bar'];
script_compiles(\@cmd, "script compiles");
script_runs(\@cmd, "script runs");
```
script_runs would fail because script_compiles() removes the first element of @cmd. So @cmd is not re-runable This is not intuitive. It's better to keep @cmd unchanged so that it can be reusable throughout a test script. 

See below places where $args is just what's passed to script_compiles()/script_runs()
https://metacpan.org/source/PLICEASE/Test-Script-1.14/lib/Test/Script.pm#L91
https://metacpan.org/source/PLICEASE/Test-Script-1.14/lib/Test/Script.pm#L148

**Comment by plicease** [comment]
Agree!

**Comment by plicease** [comment]
Thanks for reporting this I believe this to be fixed in 1.15 now on its way to CPAN.
